package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@dlsl/hardhat-deploy",
+  "version": "1.0.0",
+  "description": "This package facilitates automatic deployment and verification of Solidity Smart Contracts",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "scripts": {
+    "install": "npx husky install",
+    "lint-fix": "prettier --write \"./**/*.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/distributedlab-solidity-library/hardhat-migrate.git"
+  },
+  "keywords": [
+    "ethereum",
+    "solidity",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin",
+    "etherscan",
+    "distributedlab"
+  ],
+  "author": "Distributed Lab",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/distributedlab-solidity-library/hardhat-deploy/issues"
+  },
+  "homepage": "https://github.com/distributedlab-solidity-library/hardhat-deploy#readme",
+  "dependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.2",
+    "@truffle/deployer": "3.2.56",
+    "@truffle/reporters": "1.1.2",
+    "hardhat": "^2.12.2",
+    "lodash": "^4.17.11",
+    "web3": "^1.7.5"
+  },
+  "peerDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.2",
+    "@truffle/deployer": "3.2.56",
+    "@truffle/reporters": "1.1.2",
+    "hardhat": "^2.12.2"
```

Comment:
We only peer-depend on hardhat, other packages are included with the plugin itself

Reply:
And let the hardhat dependency be ^2.10.0